lib/api/core/plugins/pluginsManager.js

```diff
@@ -370,7 +370,7 @@ class PluginsManager {
 
       if (warnDelay) {
         pipeWarnTimer = setTimeout(() => {
-          this.trigger('log:warn', `Plugin pipe ${plugin.manifest.name}:${fn} exceeded ${warnDelay}ms to execute.`);
+          this.trigger('log:warn', `Plugin pipe ${plugin.manifest.name}:${fn} for event '${event}' exceeded ${warnDelay}ms to execute.`);
```

Comment:
This error message must be fixed: `fn` can now be a function. There are other occurences below to be fixed.